Assets/Source/LandGenerator.cs

```diff
@@ -202,9 +202,15 @@ public class LandGenerator : MonoBehaviour {
 }
 
 [System.Serializable]
+public class RPCError {
+  public string message;
+  public int code;
+}
+
+[System.Serializable]
 public class RPCResponse {
 	public string result = null;
-	public string error = null;
+  public RPCError error = null;
```

Comment:
Sorry about it, i noticed it too after I checked in. My text editor normalized and it didn't show this way for me. I will watch it.